data/human/ships.txt

```diff
@@ -1565,6 +1565,57 @@ ship "Dropship"
 
 
 
+ship "Enforcer"
+	sprite "ship/enforcer"
+	thumbnail "thumbnail/enforcer"
+	attributes
+		category "Light Warship"
+		"cost" 490000
+		"shields" 2300
+		"hull" 1100
+		"required crew" 3
+		"bunks" 5
+		"mass" 130
+		"drag" 2.9
+		"heat dissipation" .7
+		"fuel capacity" 400
+		"cargo space" 30
+		"outfit space" 210
+		"weapon capacity" 55
+		"engine capacity" 60
+		weapon
+			"blast radius" 34
+			"shield damage" 340
+			"hull damage" 170
+			"hit force" 510
+	outfits
+		"Energy Blaster" 4
+		"Laser Turret"
+		
+		"nGVF-CC Fuel Cell"
+		"LP036a Battery Pack"
+		"D14-RN Shield Generator"
+		
+		"X2700 Ion Thruster"
+		"X2200 Ion Steering"
+		"Hyperdrive"
+
+	engine 0 39
+	gun -11 -52
+	gun 11 -52
+	gun -5.5 -48
+	gun 5.5 -48
+	turret 0 -0.5
+	leak "leak" 40 30
+	leak "flame" 50 60
+	explode "tiny explosion" 8
+	explode "small explosion" 12
+	explode "medium explosion" 4
+	description "Prior to the formation of the galactic Republic, many star sectors were owned by small governing corporate entities, with the vast majority of space being inaccessible to private citizens. The only exception to this rule were the pirate bands, whose few ships were often stolen or rebellious corporate fleets of their own, yet who had an outsized impact on interstellar trade. The necessity for corporations to protect their operations meant that Enforcer was one of the more common system security vessels used for defense. The ship boasts an armament of four gun ports and a turreted mount, all packed on a small but defensible frame that was powerful for its time."
```

Comment:
```suggestion
	description `Prior to the formation of the Republic, many star systems were owned by "governing corporate entities," with the vast majority of space being inaccessible to private citizens. The only exception to this rule were the pirate bands, whose few ships were often stolen or rebellious corporate fleets of their own, yet who had an outsized impact on interstellar trade. The necessity for corporations to protect their operations meant that Enforcer was one of the most common vessels used for security and defense. The ship boasts an armament of four gun ports and a turreted mount, all packed on a small but defensible frame that was powerful for its time.`
```
Matching how the Timeline presents the "governing corporate entities." I don't see why they should be "small", though.

I think "galactic Republic" is only used in one hostile Navy hail, I was thinking of removing that mention actually. The Republic doesn't cover the whole galaxy, we don't call any governments on Earth planetary governments since they don't cover the planet.

Reply:
The phrase "galactic Republic" is used once in the historical timeline on the wiki as well. So there is precedence for it, but it's extremely rare, and the "galactic" part is always lowercased, implying it's a description rather than an official title. 

I think it actually would be fair to keep the phrase "galactic Republic" in this case. Sure, it doesn't encompass the entire galaxy, but it does encompass almost the entirety of human space, with the sparse few pirate worlds being the only planets outside of the Republic at game start, and I wouldn't be surprised if the Republic claims jurisdiction over those pirate systems, meaning that by their own description they would be a galactic government.

Reply:
Sure.

~~(The actual reason was I was trying to cut the description down.)~~